examples/src/main/scala/org/apache/spark/examples/streaming/DirectKafkaWordCount.scala

```diff
@@ -72,3 +72,4 @@ object DirectKafkaWordCount {
   }
 }
 // scalastyle:on println
+
```

Comment:
this was a dummy edit, so that i can get a clone of your branch in my Spark repo :-)